hera.rb

```diff
@@ -0,0 +1,69 @@
+class Hera < Formula
+  desc "Hera: Ewasm virtual machine conforming to the EVMC API"
+  homepage "https://github.com/ewasm/hera"
+
+  stable do
+    url "https://github.com/ewasm/hera/archive/v0.2.4.tar.gz"
+    sha256 "6d1bb30764c892f1aaada26c08add021a981d1d27638d2c9e198ca831544f9d1"
+
+    resource "cmake/cable" do
```

Comment:
Given that we're in a `stable` block, it's safe to declare these resources. When installing any formula with `--HEAD`, brew will automatically checkout git submodules, but when a stable formula is installed, getting those dependencies is up to the author.

These will only be declared on a stable install, so we can maintain them as we bump versions in the formula.